CHANGELOG.md

```diff
@@ -40,6 +40,14 @@ We no longer support link buttons being disabled or using disabled styles.
 
 This change was introduced in [pull request #3557: Remove deprecated `.govuk-button--disabled` class](https://github.com/alphagov/govuk-frontend/pull/3557).
 
+#### Remove deprecated `govuk-!-margin-static` and `govuk-!-padding-static` classes
+
+We've removed `.govuk-!-margin-static`, `.govuk-!-padding-static` and their related classes that were deprecated in [GOV.UK Frontend v4.3.1](https://github.com/alphagov/govuk-frontend/releases/tag/v4.3.1).
+
+Use the correctly named `.govuk-!-static-margin` and `.govuk-!-static-padding` classes instead.
```

Comment:
Suggesting a few tweaks just to try and mirror the language used in the original deprecation:

```suggestion
We've removed the override classes starting with `.govuk-!-margin-static` and `.govuk-!-padding-static` which were deprecated in [GOV.UK Frontend v4.3.1](https://github.com/alphagov/govuk-frontend/releases/tag/v4.3.1).

Use the classes starting with `.govuk-!-static-margin` and `.govuk-!-static-padding` instead.
```

(We're not 100% consistent whether we include the leading dot on class names – can see examples of both. Might be something for us to look at when we come to review the 'finished' release notes for v5.)

Reply:
Changelog tweaks are welcome! I was a umm-ing and aah-ing a bit over how to word it without just repeating everything from the v4.3.1 release notes.

Reply:
With regards to the dot on class names, whilst I'm thinking about it, I think it makes sense to write the names of things similarly to how they'd be written in CSS, as the syntax helps disambugate them a bit.

For example, if something says "remove the `title`" without explicitly qualifying what 'title' is, that could be interpreted as the title element, the title attribute, something with 'title' as a class name or ID, a Nunjucks macro, a parameter, or any other kind of thing.

By comparison, the intentioned meanings of `<title>`, `[title]`, `.title` and `#title` are all fairly clear to someone with some familiarity of frontend development without the need for a qualifier, which (in my mind) is the primariy audience for documents like the changelog and release notes. 

The best option is probably to do both, though!